cmd/wio/toolchain/npm/resolve/local.go

```diff
@@ -55,14 +58,19 @@ func tryFindConfig(name string, ver string, dir string) (*types.PkgConfig, error
             return nil, errors.Stringf("config %s has wrong name", paths[i])
         }
         if tryConfig.Version() != ver {
-            if i != 0 {
```

Comment:
This is needed for searching for a version in `vendor/<package_name>` as opposed to `vendor/<package_name>_<package_version>`